### DISCUSSION TITLE
How to translate?

### DISCUSSION THREAD

**Discussion by bertmelis** [immich-app/immich#386] [General]
How do you make a translation for the app?
Do I have to fork the repo and use localizely?

I'd prefer to know this up front instead of my usual "trial and error"-approach.

**Comment by alextran1502** [comment]
I only need your email and the preferred language(s) you would like to translate to, I will send the invitation email from Localizely, and you can work directly on that platform.

**Reply by bertmelis** [reply]
Dutch (but in varying degrees I speak Dutch (native), English, French, German and Polish)

e-mail: bert underscore melis at hotmail

**Reply by alextran1502** [reply]
Email sent, thank you!

**Reply by bertmelis** [reply]
Got it!

**Reply by MalteKiefer** [reply]
German, you already have my mail

**Reply by Savanta** [reply]
Polish, email: savant@fluffstuff.pl

**Reply by MalteKiefer** [reply]
@alextran1502 can you reinvite me, lost your invite

**Reply by lbandretich** [reply]
Spanish pls!
lucasrmt@gmail.com

**Reply by waclaw66** [reply]
@alextran1502: Czech, thanks!
waclaw66@seznam.cz

**Reply by alextran1502** [reply]
@lbandretich @waclaw66 invitation sent

**Reply by tugdualenligne** [reply]
French, thanks!

**Reply by Pro100x3mal** [reply]
Pro100x3mal@gmail.com
Russian

**Reply by Preventer** [reply]
w.beuve@gmail.com
Russian, thanks!

**Reply by AlexanderIlyanok** [reply]
@alextran1502
Please send invite to a.ilyanok@gmail.com, russian. Thanks.

**Reply by engels0n** [reply]
@alextran1502  Can I still participate?
engelsen@gmx.de -> german

thx in advance

**Reply by bagheriali2001** [reply]
@alextran1502 
Maybe a little bit late but can do "Persian"
bagheriali2001@gmail.com

**Reply by drebaud** [reply]
@alextran1502 
I am ready to help for french language (my mother language): denis at rebaud dot fr

**Reply by Izya12** [reply]
Please send invite to [izya12@gmail.com](mailto:izya12@gmail.com), Russian.
Thanks.

**Reply by hiagocosta** [reply]
I can help translate to PT-BR (brazil) send me an invitation hiiagocosta@gmail.com

**Comment by tugdualenligne** [comment]
Quick question: how do you enforce with docker compose, the use of a
specific language?

Le lun. 31 juil. 2023 à 07:36, Preventer ***@***.***> a
écrit :

> ***@***.***
> Russian, thanks!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/immich-app/immich/discussions/386#discussioncomment-6591167>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AD5OO4X4P4TJHP6KNT6RRPTXS475PANCNFSM542ZAKSA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
-- 
Josselin du PLESSIS | ***@***.*** | 0670810273

Envoyé depuis mon mobile, veuillez excuser brièveté et fautes d'orthographe

**Reply by Preventer** [reply]
I do not know yet. I'm waiting for the translation, then I'll figure it out.
